lib/lte_link_control/Kconfig

```diff
@@ -471,6 +494,107 @@ config LTE_LC_WORKQUEUE_STACK_SIZE
 	default 1280 if LOG_MODE_IMMEDIATE
 	default 1024
 
+# PDN (Packet Data Network) Configuration
+if LTE_LC_PDN_MODULE
+
+config LTE_LC_PDN_HEAP_SIZE
```

Comment:
If this tries to replace `HEAP_MEM_POOL_ADD_SIZE_PDN`, it cannot be named like this. It has to be strictly `HEAP_MEM_POOL_ADD_SIZE_*`. See description of HEAP_MEM_POOL_SIZE from Zephyr.